### ISSUE TITLE
less number of sequences got taxonomic assignment

### ISSUE THREAD

**Issue by rahulnccs** [benjjneb/dada2#627]
Hi, I am using A DADA2 workflow for Big Data (Paired-end). This is the snapshot output of taxa.print as you can see. I got a total of 99 ASVs in seq table and you can see most of those didn't get an assignment, is the output OK? I am using hitdb database, also tried option "tryRC=TRUE".

Thanks

[22,] NA               NA                    NA                NA                   NA               NA                           
 [23,] NA               NA                    NA                NA                   NA               NA                           
 [24,] NA               NA                    NA                NA                   NA               NA                           
 [25,] NA               NA                    NA                NA                   NA               NA                           
 [26,] NA               NA                    NA                NA                   NA               NA                           
 [27,] "Firmicutes"     NA                    NA                NA                   NA               NA                           
 [28,] NA               NA                    NA                NA                   NA               NA                           
 [29,] NA               NA                    NA                NA                   NA               NA                           
 [30,] NA               NA                    NA                NA                   NA               NA                           
 [31,] NA               NA                    NA                NA                   NA               NA                           
 [32,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Porphyromonadaceae" "Porphyromonas"  NA                           
 [33,] NA               NA                    NA                NA                   NA               NA                           
 [34,] NA               NA                    NA                NA                   NA               NA                           
 [35,] NA               NA                    NA                NA                   NA               NA                           
 [36,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Porphyromonadaceae" "Porphyromonas"  NA                           
 [37,] NA               NA                    NA                NA                   NA               NA                           
 [38,] NA               NA                    NA                NA                   NA               NA                           
 [39,] NA               NA                    NA                NA                   NA               NA                           
 [40,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Porphyromonadaceae" "Porphyromonas"  NA                           
 [41,] NA               NA                    NA                NA                   NA               NA                           
 [42,] NA               NA                    NA                NA                   NA               NA                           
 [43,] NA               NA                    NA                NA                   NA               NA                           
 [44,] NA               NA                    NA                NA                   NA               NA                           
 [45,] "Proteobacteria" "Gammaproteobacteria" "Pasteurellales"  "Pasteurellaceae"    "Haemophilus"    "Haemophilus_parainfluenzae" 
 [46,] NA               NA                    NA                NA                   NA               NA                           
 [47,] NA               NA                    NA                NA                   NA               NA                           
 [48,] NA               NA                    NA                NA                   NA               NA                           
 [49,] NA               NA                    NA                NA                   NA               NA                           
 [50,] NA               NA                    NA                NA                   NA               NA                           
 [51,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Porphyromonadaceae" "Porphyromonas"  NA                           
 [52,] NA               NA                    NA                NA                   NA               NA                           
 [53,] NA               NA                    NA                NA                   NA               NA                           
 [54,] NA               NA                    NA                NA                   NA               NA                           
 [55,] NA               NA                    NA                NA                   NA               NA                           
 [56,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Prevotellaceae"     "Prevotella"     NA                           
 [57,] "Firmicutes"     "Bacilli"             "Lactobacillales" "Streptococcaceae"   "Streptococcus"  NA                           
 [58,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Porphyromonadaceae" "Porphyromonas"  NA                           
 [59,] "Firmicutes"     "Bacilli"             "Gemellales"      "Gemellaceae"        "Gemella"        NA                           
 [60,] "Proteobacteria" "Gammaproteobacteria" "Pasteurellales"  "Pasteurellaceae"    "Haemophilus"    "Haemophilus_parainfluenzae" 
 [61,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Prevotellaceae"     "Prevotella"     "Prevotella_veroralis"       
 [62,] "Firmicutes"     NA                    NA                NA                   NA               NA                           
 [63,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Prevotellaceae"     "Prevotella"     "Prevotella_melaninogenica"  
 [64,] "Firmicutes"     "Bacilli"             "Gemellales"      "Gemellaceae"        "Gemella"        "Gemella_haemolysans"        
 [65,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Prevotellaceae"     "Prevotella"     "Prevotella_melaninogenica"  
 [66,] "Fusobacteria"   "Fusobacteriia"       "Fusobacteriales" "Leptotrichiaceae"   "Leptotrichia"   "Leptotrichia_buccalis"      
 [67,] "Bacteroidetes"  "Bacteroidia"         "Bacteroidales"   "Prevotellaceae"     "Prevotella"     "Prevotella_veroralis"

**Comment by benjjneb** [comment]
If you try using a more universal database, like Silva (again with `tryRC=TRUE`) what assignments are made to those `NA NA NA NA NA NA` taxa?

If still seeing mostly `NA`s, I would try BLAST-ing a few against nt and seeing what they hit (or don't?) to try an even broader database. That information will help a lot in diagnosing what is going on.

**Comment by rahulnccs** [comment]
Yes, I did BLAST, also tried tax-assignment with SILVA DB, those ASVs were not related to any of the bacterial taxa, I am good to go!
 But still, why only 101 ASVs(taxa) in seqtab?
 Its an Illumina generated data. Previously, I got around 1100 ASVs for one of the Illumina-Hiseq run. The probable reason might be my sampling site, this is sequencing data for oral samples, that harbors less microbial diversity as compared to the gut. 
Thanks.
